tests/server/blueprints/panels/test_panels_views.py

```diff
@@ -155,22 +155,27 @@ def test_panel_export(client, real_panel_database):
     assert resp.status_code == 200
 
 
-def test_panel_export_case_hits(client, real_panel_database):
+def test_panel_export_case_hits(app, real_panel_database):
     """Test function used for exporting all genes with variant hits for a case"""
 
     # GIVEN a case and a gene panel in the database
     adapter = real_panel_database
     panel_obj = adapter.panel_collection.find_one()
     case_obj = adapter.case_collection.find_one()
 
-    # WHEN downloading the case variants hits report
-    form_data = {"case_name": " - ".join([case_obj["owner"], case_obj["display_name"]])}
-    resp = client.post(url_for("panels.panel_export", panel_id=panel_obj["_id"]), data=form_data)
+    # GIVEN an initialized app and a valid user
```

Comment:
When you invoke the panel_export_case_hits endpoint you need to be in the app context, because of [this line](https://github.com/Clinical-Genomics/scout/blob/f7bc607afb50749b785f6fbce2ddcceed7080415/scout/server/blueprints/panels/views.py#L295)

Reply:
From the function name it very much seems to have been the intention to test this endpoint. Good catch!